packages/legend-application-studio/src/stores/EditorStore.ts

```diff
@@ -438,9 +438,10 @@ export class EditorStore implements CommandRegistrar {
       key: LEGEND_STUDIO_COMMAND_KEY.TOGGLE_TEXT_MODE,
       trigger: this.createEditorCommandTrigger(
         () =>
-          this.isInitialized &&
-          (!this.isInConflictResolutionMode ||
-            this.conflictResolutionState.hasResolvedAllConflicts),
+          this.isInViewerMode ||
```

Comment:
why arent the other conditions meet here. I would expect the flag this.isInitialized to be true in view mode once we finish building the graph. 
Are the conflicts conditions not met?

Reply:
Changed `isInitialized` method instead to add condition of `systemBuildState.hasSucceeded` if in viewer mode 👍